bindings/python/tests/bindings/test_normalizers.py

```diff
@@ -75,6 +75,8 @@ def test_items(self):
         assert normalizers[2].__class__ == Strip
         with pytest.raises(IndexError):
             print(normalizers[3])
+        normalizers[1] = Strip()
+        assert normalizers[1].__class__ == Strip
 
```

Comment:
let's check set_item and also that get_item_ is mutabkle

Reply:
It isn't atm, need to impl getters and setters for each variant of the Normalizer/Pretok/Processor. Will do in a subsequent PR.

Reply:
okay!